SQF/dayz_code/actions/plotManagement/plotAddFriend.sqf

```diff
@@ -15,7 +15,7 @@ _friends = _thePlot getVariable ["plotfriends",[]];
 _toAdd = [_friendUID,toArray _friendName];
 _inList = false;
 
-{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList = true; }; } count _friends;
+{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList = true; }; false } count _friends;
```

Comment:
Whenever you use the { } loop syntax of the count command the script expects the final return to be a bool. In this case it should not be necessary because the exitWith should always run, so it will return a bool.

Here is an example of where it was returning a number:
https://github.com/oiad/DayZ-Epoch/blob/bc7c1879a328f54eb2686f0ee2d5120499604537/SQF/dayz_code/actions/plotManagement/plotNearbyHumans.sqf#L6-L13

The last command that runs in the count loop was lbAdd which returns a number. That would cause the error "Type number expected bool". I will add a note on the BI wiki page about that because it has probably caused a lot of confusion for people (myself included).